### PR TITLE
DEV-3517-feat(pisicine-go): add README.md for new exercise shoppinglistsort

### PR DESCRIPTION
**Pull request by tiagocollot**
Created the README.md for the new go exercise shoppinglistsort

### CODE REVIEW

File: subjects/shoppinglistsort/README.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# shoppinglistsort
+
+### Instructions
+
+You were sent to the supermarket with a shopping list. To make your shopping faster, write a function `ShoppingListSort()` that takes an array of strings and sorts it, according to the string length, returning an array in which the strings appear in ascending order.
+
+In the case of equal string length the order does not matter.
```
Comment (jotapero) [inline subjects/shoppinglistsort/README.md:7]: sugestion to rephrase as: "When the length of string elements are equal, then the order in which they appear does not matter"

**Review by jotapero — CHANGES_REQUESTED** [review]

File: subjects/shoppinglistsort/README.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
+# shoppinglistsort
+
+### Instructions
+
+You were sent to the supermarket with a shopping list. To make your shopping faster, write a function `ShoppingListSort()` that takes an array of strings and sorts it, according to the string length, returning an array in which the strings appear in ascending order.
```
Comment (jotapero) [inline subjects/shoppinglistsort/README.md:5]: To keep robustness like the platform " (...) write a function ~ShoppingListSort()~ that takes and array of strings (...) "

**Review by jotapero — CHANGES_REQUESTED** [review]

File: subjects/shoppinglistsort/README.md (hunk 1 above)
Comment (tiagocollot) [reply subjects/shoppinglistsort/README.md]: Great suggestion! Done!

**Review by jotapero — APPROVED** [review]
Everything in order

**Review by jotapero — APPROVED** [review]

File: subjects/shoppinglistsort/README.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
+## shoppinglistsort
+
+### Instructions
+
+You were sent to the supermarket with a shopping list. To make your shopping faster, write a function `ShoppingListSort()` that takes an array of strings and sorts it, according to the string length, returning an array in which the strings appear in ascending order.
```
[the remark this answers is not in the document]
Comment (tiagocollot) [reply subjects/shoppinglistsort/README.md:5]: It is fixed.

File: subjects/shoppinglistsort/README.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+
+When the length of string elements are equal, then the order in which they appear does not matter.
+
+### Expected function
+
+```go
+func ShoppingListSort(array[] string)[]string {
```
[the remark this answers is not in the document]
Comment (tiagocollot) [reply subjects/shoppinglistsort/README.md:12]: Thank you for noticing, it is fixed.

File: subjects/shoppinglistsort/README.md [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+
+func main() {
+  array:= []string{"Banana", "Mushroom", "Salt", "Pepper","Tea", "Milk"}
+  fmt.Println(piscine.ShoppingListSort(array))
+}
+```
+
+And its output:
+
+```go
```
[the remark this answers is not in the document]
Comment (tiagocollot) [reply subjects/shoppinglistsort/README.md:37]: Thank you, it is changed.

File: subjects/shoppinglistsort/README.md [hunk 6]
```diff
@@ -0,0 +1,40 @@
+## shoppinglistsort
+
+### Instructions
+
+You were sent to the supermarket with a shopping list. To make your shopping faster, write a function `ShoppingListSort()` that takes an array of strings and sorts it, according to the string length, returning an array in which the strings appear in ascending order.
+
+When the length of string elements are equal, then the order in which they appear does not matter.
```
[the remark this answers is not in the document]
Comment (eslopfer) [reply subjects/shoppinglistsort/README.md:7]: This was not addressed.

File: subjects/shoppinglistsort/README.md [hunk 7]
```diff
@@ -4,7 +4,7 @@
 
 You were sent to the supermarket with a shopping list. To make your shopping faster, write a function `ShoppingListSort()` that takes a slice of strings and sorts it, according to the string length, returning a slice in which the strings appear in ascending order.
 
-When the length of string elements are equal, then the order in which they appear does not matter.
+- The length of the strings cannot repeat itself.
```
Comment (eslopfer) [inline subjects/shoppinglistsort/README.md:7]: Wouldn't it be better to say "The lengths of the strings being tested will be different". Feel free to resolve the conversation if you disagree.

**Review by eslopfer — APPROVED** [review]
LGTM! :rocket:

**Pull request merged**